Backend/BackendAPI/Data/CommonDataContext.cs

```diff
@@ -1,33 +1,26 @@
 ﻿using BackendAPI.Models;
-using BackendAPI.ModelsConfigurations;
 using Microsoft.EntityFrameworkCore;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace BackendAPI.Data
 {
-    public class DataContext : DbContext
+    //Klasa opisująca wspólną część contextów dla różnych środowisk
+    //Ma tutaj byc definicja DbSetów.
+    public class CommonDataContext : DbContext
     {
-        public DataContext(DbContextOptions<DataContext> 
-            options)
-           : base(options)
-        {
-        }
-
         public DbSet<User> Users { get; set; }
         public DbSet<Bike> Bikes { get; set; }
         public DbSet<BikeStation> BikeStations { get; set; }
         public DbSet<Rental> Rentals { get; set; }
 
+        public CommonDataContext(DbContextOptions<CommonDataContext>
+            options)
+           : base(options)
+        {
+        }
+
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             base.OnModelCreating(modelBuilder);
-            modelBuilder.ApplyConfiguration(new UserConfiguration());
-            modelBuilder.ApplyConfiguration(new BikeConfiguration());
-            modelBuilder.ApplyConfiguration(new BikeStationConfiguration());
-            modelBuilder.ApplyConfiguration(new RentalConfiguration());
         }
```

Comment:
to można wywalić chyba

Reply:
to musi zostac, bo trzeba wywolac OnModelCreating z DbContext, a Production i test nie dziedziczy DbContext